src/mbgl/style/expression/interpolate.cpp

```diff
@@ -216,6 +216,33 @@ std::vector<optional<Value>> InterpolateBase::possibleOutputs() const {
     return result;
 }
 
+template <typename T>
+mbgl::Value Interpolate<T>::serialize() const {
+    static const std::string interpolate = "interpolate";
+    static const std::string exponentialTag = "exponential";
+    static const std::string cubicBezierTag = "cubic-bezier";
+    std::vector<mbgl::Value> serialized;
+    serialized.emplace_back(interpolate);
+    if (interpolator.is<ExponentialInterpolator>()) {
+        const ExponentialInterpolator& exponential = interpolator.get<ExponentialInterpolator>();
+        // TODO: Could do a separate serialization for "linear", but probably no need?
+        serialized.emplace_back(std::vector<mbgl::Value>{{ exponentialTag, exponential.base }});
```

Comment:
Is there any downside to just saying `std::vector<mbgl::Value>{{ "exponential", exponential.base }}`?  The string's gotta get copied into the vector anyway, right?

Reply:
The strings are really just there to help the `std::vector<mbgl::Value>` constructor deduce the type -- a plain char* is ambiguous for it.